GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -951,7 +1060,8 @@ private String sanitize(String s) {
                                         + " - " + level;
                                 createCell(row, offset++, levelName, headerStyle);
                             }
-                        } else {
+                        }
+                        else {
```

Comment:
perhaps for readability it could be an idea to create another `if` branch for `OPTION` questions just like we do for `GEO` and `CASCADE` on lines 1029 & 1054 respectively.